### PR TITLE
Add db upgrader from ent 3.x to 3.4

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

**Review by cangfengzhs — APPROVED** [review]

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# 升级企业版 NebulaGraph 3.x 至 3.4.0  
+
+本文以升级企业版 NebulaGraph 3.1.0 为例，介绍如何升级企业版 NebulaGraph 3.x 至 3.4.0。
```
Comment (cooper-lzy) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md:3]:
```suggestion
本文以企业版 NebulaGraph 3.1.0 为例，介绍如何升级至 3.4.0。
```

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## 升级说明
+
+- 此升级操作仅适用于升级企业版 NebulaGraph 3.x 至 3.4.0。<!--因为企业版 NebulaGraph 3.4 中一个分片对应一个 RocksDB 实例不同于 3.4 之前的一个图空间对应一个 RocksDB 实例。--> 如果您的企业版 NebulaGraph 版本低于 3.x，请先升级到企业版 3.x 再升级到 3.4.0。具体操作请参见[升级企业版 NebulaGraph 2.x 至 3.x](upgrade-nebula-graph-to-latest.md)。
+
+- 仅支持单机的升级操作，即执行升级操作的集群 IP 地址必须与原集群相同。
```
Comment (cooper-lzy) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md:9]:
```suggestion
- 执行升级操作的集群 IP 地址不能变更。
```

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  
+- 机器硬盘剩余空间至少需为原数据目录的 1.5 倍。
+  
+
+## 升级步骤
+
+1. [联系我们获取](https://www.nebula-graph.com.cn/contact)并安装企业版 NebulaGraph v3.4.0 的安装装包。
```
Comment (cooper-lzy) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md:16]:
```suggestion
1. [联系我们获取](https://www.nebula-graph.com.cn/contact)企业版 NebulaGraph v3.4.0 的安装包并安装。
```

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+   
+  !!! note
+
+        不同安装包的升级步骤相同。本文以 RPM 包且安装目录为`/usr/local/nebulagraph-ent-3.4`为例。具体操作请参见[安装 RPM 包](../2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md)。 
+   
+  !!! caution
+
+        请确保 3.4.0 集群的 meta 服务和 storage 服务的配置文件中的`--data_path`参数的值与 3.x 集群的配置文件中的`--data_path`参数的值相同。否则，升级后的集群无法启动。
```
Comment (cooper-lzy) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md:24]:
```suggestion
        请确保 3.4.0 集群的 Meta 服务和 Storage 服务的配置文件中的`--data_path`参数的值与 3.x 集群的配置文件中的`--data_path`参数的值相同。否则，升级后的集群无法启动。
```

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+    ```bash
+    sudo ./bin/meta_upgrader --src_meta_path=/usr/local/nebula-ent-3.1.0/data/meta --dst_meta_path=/usr/local/nebula-ent-3.4/data/meta
+    ```
+
+    如果有多个源 Meta 数据目录，请指定不同的源 Meta 数据目录和目标 Meta 数据目录并分别执行命令。
+
+  服务升级完成后，会在 v3.4.0 的安装目录下生成`data`目录，其中包含升级后的数据文件。
+
+1. 上传 License 文件至 v3.4.0 的安装目录下的`share/resources`内。
```
Comment (cooper-lzy) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md:82]:
```suggestion
4. 上传 License 文件至 v3.4.0 的安装目录下的`share/resources`内。
```

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+2. 启动和连接企业版 NebulaGraph v3.4.0 服务后，验证数据是否正确。参考命令如下：
```
Comment (cooper-lzy) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md:84]:
```suggestion
5. 启动和连接企业版 NebulaGraph v3.4.0 服务后，验证数据是否正确。参考命令如下：
```

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
+# 升级企业版 NebulaGraph 3.x 至 3.4.0  
+
+本文以升级企业版 NebulaGraph 3.1.0 为例，介绍如何升级企业版 NebulaGraph 3.x 至 3.4.0。
+
+## 升级说明
+
+- 此升级操作仅适用于升级企业版 NebulaGraph 3.x 至 3.4.0。<!--因为企业版 NebulaGraph 3.4 中一个分片对应一个 RocksDB 实例不同于 3.4 之前的一个图空间对应一个 RocksDB 实例。--> 如果您的企业版 NebulaGraph 版本低于 3.x，请先升级到企业版 3.x 再升级到 3.4.0。具体操作请参见[升级企业版 NebulaGraph 2.x 至 3.x](upgrade-nebula-graph-to-latest.md)。
```
Comment (randomJoe211) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-ent-from-3.x-3.4.md:7]:
```suggestion
- 此升级操作仅适用于升级企业版 NebulaGraph 3.x 至 3.4.0。<!--因为企业版 NebulaGraph 3.4 中一个分片对应一个 RocksDB 实例不同于 3.4 之前的一个图空间对应一个 RocksDB 实例。--> 如果您的企业版 NebulaGraph 版本低于 3.0.0，请先升级到企业版 3.x 再升级到 3.4.0。具体操作请参见[升级企业版 NebulaGraph 2.x 至 3.x](upgrade-nebula-graph-to-latest.md)。
```
3.x在这里有歧义，低于3.2也是低于3.x

**Review by whitewum — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**